plugins/artifacts/config.go

```diff
@@ -0,0 +1,61 @@
+package artifacts
+
+import (
+	schematypes "github.com/taskcluster/go-schematypes"
+	"github.com/taskcluster/taskcluster-worker/runtime/util"
+)
+
+type config struct {
+	PrivateKey         string `json:"privateKey"`
+	AlwaysCreateCOT    bool   `json:"alwaysCreateCOT"`
```

Comment:
Is there a real use case scenario for this config? If it is a "nice to have" feature, I am in favor of removing it.

Reply:
Not sure, I think we'll just end up setting the config to the default behavior we want then never touch it again...

Reply:
My vote is to remove it, let's not add code candidate to be obsolete :)

Reply:
okay, let's remove it...